services/cinema_hall.py

```diff
@@ -0,0 +1,12 @@
+from django.db.models import QuerySet
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> QuerySet:
+    return CinemaHall.objects.all()
+
+
+def create_cinema_hall(hall_name: str, hall_rows: int,
+                       hall_seats_in_row: int) -> None:
```

Comment:
The same moments like were in my code. Function should not return None.
It's better to return object.